projects/vdk-plugins/vdk-notebook/src/vdk/plugin/notebook/notebook_plugin.py

```diff
@@ -22,6 +26,22 @@ def initialize_job(self, context: JobContext):
             for file_path in notebook_files:
                 Notebook.register_notebook_steps(file_path, context)
 
+    @hookimpl(hookwrapper=True)
+    def run_job(self, context: JobContext) -> None:
+        out: HookCallResult
+        out = yield
+        result: ExecutionResult = out.get_result()
+        step_results = result.__dict__["steps_list"]
+        for step_result in step_results:
+            if step_result.__dict__["status"] == ExecutionStatus.ERROR:
```

Comment:
why the use of `__dict__` ? result.steps_list ? and step_result.status

Reply:
I get error "AttributeError: 'ExecutionResult' object has no attribute 'steps_lists'"

Reply:
That doesn't make sense, you can see that right? Here is the definition of the class : https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-core/src/vdk/internal/builtin_plugins/run/execution_results.py#L46 

Clearly, the attribute is there. Please, If something doesn't make sense you really need to work harder to understand why it doesn't make sense. 

Hmmm,  if the error message you pasted is correct, the error is due to a typo in the attribute name (the error message says "**steps_lists**")  .  you You should set it to **steps_list** (without the 's' at the end).

Reply:
fixed